pyproject.toml

```diff
@@ -101,25 +102,36 @@ ignore = [
     "D400",
 ]
 select = [
-    "E",  # pycodestyle
-    "F",  # pycodestyle
-    "N",  # pep8-naming
-    "W",  # pycodestyle
-    "D",  # pydocstyle
+    "E",       # pycodestyle
+    "F",       # pycodestyle
+    "N",       # pep8-naming
+    "W",       # pycodestyle
+    "D",       # pydocstyle
+    "UP",      # pyupgrade
+    "I",       # isort
+    "RUF022",  # sort __all__
```

Comment:
Consider also using "C4" and some subset of "B".

Reply:
I just copied these settings from `daf_butler`.

Maybe add these per your suggestion?

```
    "C4",      # flake8-comprehensions
    "B",       # flake8-bugbear
```